CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### Version 1.4.0
```

Comment:
You shouldn't update the changelog and the version, it's done automatically via [bump](https://github.com/fabiospampinato/bump).

Reply:
Oh Ok.